silk/sql.py

```diff
@@ -59,18 +59,7 @@ def _explain_query(connection, q, params):
         else:
             prefixed_query = f"{prefix} {q}"
         with connection.cursor() as cur:
-            try:
-                params_str = tuple(force_str(param) for param in params)
-            except UnicodeDecodeError:
-                # Sometimes `force_str` can still raise a UnicodeDecodeError
-                # Reference: https://github.com/jazzband/django-silk/issues?q=encoding
-                Logger.error(
-                    "UnicodeDecodeError while trying to explain query: %s. "
-                    "This could be caused by a non-UTF-8 encoded parameter.",
-                    q
-                )
-                return None
-            cur.execute(prefixed_query, params_str)
+            cur.execute(prefixed_query, params)
             result = _unpack_explanation(cur.fetchall())
             return '\n'.join(result)
```

Comment:
It might be that *these* lines are actually the part that's problematic for non-UTF-8 data? Looking at #798 and specifically the initial commit of 584a6e2fe36e3d8234dc30b45f376f440d4dde38, it was wrapping all three lines here in the try/except at first.

Reply:
Looking at #819 , the error people are getting seem to be an error from the postgresql database itself.  That would only happen in the `cur.execute()` so the exception happens before these two lines are even reached.